unittests/CppInterOp/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ export_executable_symbols(CppInterOpTests)
 
 unset(LLVM_LINK_COMPONENTS)
 
-add_cppinterop_unittest(DynamicLibraryManagerTests DynamicLibraryManagerTest.cpp)
+add_cppinterop_unittest(DynamicLibraryManagerTests DynamicLibraryManagerTest.cpp ${CMAKE_SOURCE_DIR}/lib/Interpreter/Paths.cpp)
```

Comment:
Why did we add this here? That should be provided by the shared object file.

Reply:
It was because it wasn't getting linked on one of the platforms

Reply:
PTAL https://github.com/compiler-research/CppInterOp/actions/runs/7944478504/job/21690038529?pr=193#step:20:172 @vgvassilev